event.go

```diff
@@ -52,10 +53,7 @@ func newEventMessage(i interface{}) *EventMessage {
 
 // ToString format message to string
 func (p *EventMessage) ToString() (string, error) {
```

Comment:
The signature for a function implementing the Stringer interface is `String() string`, not `ToString() (string, error)`